### PR TITLE
Prep for CI

### PR DESCRIPTION
**Pull request by zaro0508**
* Update to use latest ver of Android studio 3.0
* Make gradlew and gradlew.bat executable files

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ buildscript {
     }
     dependencies {
         // wasn't working with travis
```
Comment (zaro0508) [inline build.gradle]: hmm.  what's up with this comment?

File: .idea/modules.xml [hunk 2]
```diff
@@ -4,6 +4,7 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/CardiorespiratoryFitness-Android.iml" filepath="$PROJECT_DIR$/CardiorespiratoryFitness-Android.iml" />
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
```
Comment (zaro0508) [inline .idea/modules.xml]: This was auto added by android studio not sure why it added a duplicate line.  I'm wondering whether this file should be in .gitignore?

Comment (liujoshua) [reply .idea/modules.xml]: It doesn't seem to hurt to add modules to .gitignore (other than maybe AndroidStudio first load) but this is supposedly one of the /.idea/ files that can be checked into source control

File: build.gradle (hunk 1 above)
Comment (liujoshua) [reply build.gradle]: Please remove. This was when the beta build tool wasn't available in Travis and I was falling back to an old version

**Pull request merged**